pkg/kubelet/kubelet_pods.go

```diff
@@ -944,6 +944,24 @@ func (kl *Kubelet) PodResourcesAreReclaimed(pod *v1.Pod, status v1.PodStatus) bo
 		klog.V(3).Infof("Pod %q is terminated, but some containers have not been cleaned up: %s", format.Pod(pod), statusStr)
 		return false
 	}
+	// pod's sandboxes should be deleted
+	filter := &runtimeapi.PodSandboxFilter{
+		LabelSelector: map[string]string{kubetypes.KubernetesPodUIDLabel: string(pod.UID)},
+	}
+	sandboxes, err := kl.runtimeService.ListPodSandbox(filter)
```

Comment:
Can we get this form the runtimeStatus above?  Or is it necessary to make an additional CRI call here?

Reply:
Yes, we can get them from the current runtime implementation but then i saw the comment https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/container/runtime.go#L283-L284 and hence decided against it

Reply:
That comment is from the pre-CRI days when we had multiple in-tree container runtimes (e.g. rkt).  We should be able to use that field for our purpose here.

Reply:
good to know that.I will use that and shall i also remove that comment because it is confusing for people who doesn't have history?

Reply:
yeah, go ahead and remove the comment.